Sources/Dollar.swift

```diff
@@ -1236,6 +1236,22 @@ open class `$` {
         return array.count
     }
 
+    /// Return the sum of an array of integer numbers
+    ///
+    /// - parameter array: The array to source from.
+    /// - returns: Sum of all int of the array
+    open class func sum(_ array: [Int]) -> Int {
```

Comment:
You can make this generic type such that it implements `IntegerArithmeticType` so you dont have to add Int and Double or Float, etc.

```swift
open class func sum<T: IntegerArithmeticType>(_ array: [T]) -> T {
```